cider-dap/src/client.rs

```diff
@@ -0,0 +1,105 @@
+use std::io::{BufWriter, Write};
+use std::net::TcpStream;
+
```

Comment:
This file is no longer needed due to the changes in the dap crate set up. As
such it should be fine to delete it